### ISSUE TITLE
zombie process of cpptools-srv is created continuously

### ISSUE THREAD

**Issue by seokbeomKim** [microsoft/vscode-cpptools#9261]
### Environment

- OS and version: Windows
- VS Code: 1.66.2
- C/C++ extension: v1.10.0
- OS and version of remote machine (if applicable): Fedora Linux / 3.19.8-100.fc20.x86_64 (remote server)
- GDB / LLDB version: Unknown

### Bug Summary and Steps to Reproduce

Bug Summary: Zombie process of 'cpptools-srv <defunct>' is created continuously.

Steps to reproduce:
1. Use the c/cpp Intellisense extension in a remote server.
2. When I check the process list, the message is like (and the number of zombie processes keeps increasing):
```
  PID TTY          TIME CMD
  622 ?        00:00:00 sshd
  623 pts/3    00:00:00 sh
 2645 pts/1    00:00:00 bash
 6871 ?        00:00:01 sshd
 6872 ?        00:00:00 bash
 6913 ?        00:00:00 sh
 6925 ?        00:00:12 node
 6969 ?        00:00:02 node
 7031 ?        00:05:37 node
 7042 ?        00:00:02 node
 7144 ?        00:05:18 cpptools
 7259 ?        00:00:06 vsls-agent
 7267 ?        00:00:00 node
 7496 ?        00:42:00 KnottyUI
 7589 ?        00:00:01 cpptools-srv <defunct>
 7631 ?        00:01:03 node
 7748 ?        00:00:00 cpptools-srv <defunct>
 7900 ?        00:00:00 cpptools-srv <defunct>
 7918 ?        00:38:37 KnottyUI
 9839 pts/0    00:00:01 bash
10573 pts/4    00:00:00 bash
11056 ?        00:00:00 cpptools-srv <defunct>
11293 ?        00:00:00 cpptools-srv <defunct>
11489 ?        00:00:01 sshd
11490 pts/26   00:00:00 bash
11537 pts/26   00:00:00 tmux
11546 pts/27   00:00:00 bash
12993 pts/14   00:00:00 bash
14605 ?        00:00:00 cpptools-srv <defunct>
14694 ?        00:00:00 cpptools-srv <defunct>
14980 ?        00:00:00 cpptools-srv <defunct>
15018 ?        00:00:00 cpptools-srv <defunct>
15054 ?        00:00:00 cpptools-srv <defunct>
15135 pts/1    00:00:00 vim
15143 ?        00:00:00 cpptools-srv
16249 pts/12   00:00:00 bash
19963 ?        00:00:00 sleep
20126 pts/27   00:00:00 ps
28678 ?        02:11:56 tmux
B
```

3. This bug results in the situation that I cannot login to the server with the following message unless I kill the zombie processes named 'cpptools-srv':
```
ssh myuser@192.168.33.101
shell request failed on channel 0
```

### Debugger Configurations

```shell
I enabled error log of c/c++ extension as debug:

"C_Cpp.loggingLevel": "Debug"
```

### Debugger Logs

```shell
Queueing IntelliSense update for files in translation unit of: /home/ ...(removed due to private data)
cpptools/getCodeActions: /home/(removed due to private data) (id: 26)
cpptools/finishUpdateSquiggles
Error squiggle count: 7

-> It seems that zombie process is created whenever printing out the error message (Error squiggle count: ).
```

**Comment by Colengms** [comment]
Hi @seokbeomKim .  Thanks for reporting this.  I believe I can reproduce the problem.  It appears to be a recent regression in 1.10.0.  You should be able to work around the issue by reverting to 1.9.8, for now.

**Comment by seokbeomKim** [comment]
@Colengms Thanks for the confirmation. I will revert to version 1.9.8 temporarily.

**Comment by sean-mcmanus** [comment]
This is fixed with 1.10.2 (pre-release).
